src/photosList/index.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+
+const useFetchedPhotos = (userId) => {
+  const [photos, setPhotos] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    if (userId) {
```

Comment:
If the `userId` is falsy then there is no `setIsLoading(false);`, which causes infinite Loading

Reply:
done